tdei-api-gateway.json

```diff
@@ -432,253 +468,208 @@
                 ]
             }
         },
-        "/api/v1/gtfs-flex": {
-            "get": {
+        "/api/v1/osw/format/upload": {
+            "post": {
                 "tags": [
-                    "GTFS-Flex"
+                    "OSW"
                 ],
-                "summary": "List flex files meeting specified criteria.",
-                "description": "This endpoint returns a json list of all gtfs flex files stored in the TDEI system that meet the specified criteria. Criteria that can be specified include: polygon (bounding box), minimum confidence level, flex version, date time and project group id.\n\nThis endpoint can be used by an application developer to obtain a list of gtfs flex files in the TDEI system meeting the specified criteria. This endpoint returns a list of file-metadata including the uris of the file, which can be used to fetch the files themselves.",
-                "operationId": "listFlexFiles",
-                "parameters": [
-                    {
-                        "name": "tdei_service_id",
-                        "in": "query",
-                        "description": "Id of the flex service.",
-                        "required": false,
-                        "schema": {
-                            "type": "string"
-                        }
-                    },
-                    {
-                        "name": "bbox",
-                        "in": "query",
-                        "description": "A bounding box which specifies the area to be searched. A bounding box is specified by a string providing the lat/lon coordinates of the corners of the bounding box. Coordinate should be specified as west, south, east, north.",
-                        "required": false,
-                        "schema": {
-                            "maxItems": 4,
-                            "minItems": 4,
-                            "type": "array",
-                            "items": {
-                                "type": "number",
-                                "format": "double"
+                "summary": "Request osw format on demand",
+                "description": "Upload a file and request for formatting on demand",
```

Comment:
upload a file and request for file format conversion

Reply:
Done